.changeset/afraid-books-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+'vite-imagetools': patch
+---
+
+fix: use URI-decoded pathname when emitting file
```

Comment:
can we split the URI decoding stuff out into a separate PR?

Reply:
Yes, but it will look different before and after caching. Do you want the URI decoding before the cache is integrated or after?

Reply:
I created PR #689 that can be applied before the caching is merged.
If you decide to merge this PR first, I will rebase the cache PR and add the necessary changes.
If you decide to merge the cache PR first, I will rebase this PR and add the necessary changes.